From 62f2f9c3a01bfee46f20ce0d78f515a711f214fa Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Mon, 14 Nov 2022 09:43:19 +0100
Subject: [PATCH 120/132] Revert
 "maple_tree-reorganize-testing-to-restore-module-testing-fix-2"

This reverts commit 398d33b8a28b65ed7599f2f1cac64d5e1c81308a.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 lib/maple_tree.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index 4577b6de6..0227eef7b 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -6019,7 +6019,6 @@ void *mas_find(struct ma_state *mas, unsigned long max)
 	/* Retries on dead nodes handled by mas_next_entry */
 	return mas_next_entry(mas, max);
 }
-EXPORT_SYMBOL_GPL(mas_find);
 
 /**
  * mas_find_rev: On the first call, find the first non-null entry at or below
-- 
2.38.1.473.ga0789512c5

